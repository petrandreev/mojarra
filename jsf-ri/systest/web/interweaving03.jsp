<!--
Copyright 2004 Sun Microsystems, Inc. All rights reserved.
SUN PROPRIETARY/CONFIDENTIAL. Use is subject to license terms.
-->
<%@ page contentType="text/html" language="java" %>
<%@ taglib prefix="c" uri="http://java.sun.com/jstl/core" %>
<%@ taglib prefix="f" uri="http://java.sun.com/jsf/core" %>
<%@ taglib prefix="h" uri="http://java.sun.com/jsf/html" %>

<f:view>
    <h:form>
        <h:panelGrid columns="1">
            Row 1
            <h:outputText value="Row 2"/>
            Row 3
        </h:panelGrid>
    </h:form>
</f:view>
