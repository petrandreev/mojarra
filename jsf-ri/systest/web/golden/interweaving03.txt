<!--
 Copyright 2004 Sun Microsystems, Inc. All rights reserved.
 SUN PROPRIETARY/CONFIDENTIAL. Use is subject to license terms.
-->





<form id="j_id_id10" name="j_id_id10" method="post" action="/jsf-systest/faces/interweaving03.jsp;jsessionid=D623839373C2B21D75B00AA4D67477BA" enctype="application/x-www-form-urlencoded">
<table>
<tbody>
<tr>
<td>
	Row 1
	</td>
</tr>
<tr>
<td>Row 2</td>
</tr>
<tr>
<td>
	Row 3
      </td>
</tr>
</tbody>
</table>
<input type="hidden" name="javax.faces.ViewState" id="javax.faces.ViewState" value="_id5:_id6" /><input type="hidden" name="j_id_id10" value="j_id_id10" /></form>
